x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table.tsx

```diff
@@ -5,22 +5,20 @@
  * 2.0.
  */
 
-import React, { useCallback } from 'react';
+import React, { useCallback, useEffect } from 'react';
 import { EuiInMemoryTable } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { isEqual } from 'lodash';
+import { SnapshotMetricType } from '../../../../../common/inventory_models/types';
```

Comment:
Nit: I think this can stay as `import type` or?

Reply:
Absolutely! Thanks for spotting this.